src/main/java/dev/codesupport/web/common/security/AuthenticationRequest.java

```diff
@@ -0,0 +1,20 @@
+package dev.codesupport.web.common.security;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.io.Serializable;
+
+/**
+ * Model for incoming authorization requests.
+ */
+@Data
+@NoArgsConstructor
+public class AuthenticationRequest implements Serializable {
+
+    private static final long serialVersionUID = 4690455532399413757L;
```

Comment:
Why is this hardcoded? What does it do?

Reply:
serialVersionUID? it's something java uses for serialization in order to validate that the class in the classLoader is the same version as the object it's deserializing.  If you don't specify it, it will automatically generate one, but that can create issues.

Reply:
Gotcha.